src/games/strategy/engine/framework/startup/ui/GameSelectorPanel.java

```diff
@@ -311,7 +343,7 @@ public void run() {
       m_gameOptions.setEnabled(false);
     }
     // we don't want them starting new games if we are an old jar
-    if (GameRunner2.areWeOldExtraJar()) {
+    if (ClientFileSystemHelper.areWeOldExtraJar()) {
```

Comment:
From here the rest of the changes are standard method moves. Followed by that are updated test cases and new ones.